Update azure-config-guide-for-cloud-siem.md (#33085)
DataDog/documentation: content/en/security/cloud_siem/guide/azure-config-guide-for-cloud-siem.md

### Add the Datadog Azure function

> <br />**Note**: The steps described below are based on the Azure Functions Programming Model v3. Datadog’s latest [Azure function code][8] has been updated to support the Azure Functions Programming Model v4, and the documentation is currently being revised to reflect these changes. If you need guidance on deploying the v4-compatible version or have questions regarding the differences between v3 and v4, please contact Datadog Technical Support for assistance.

Reviewer: ```suggestion
<br />**Note**: The steps described below are based on the Azure Functions Programming Model v3. Datadog's latest [Azure function code][8] has been updated to support the Azure Functions Programming Model v4. If you need guidance on deploying the v4-compatible version or have questions regarding the differences between v3 and v4, contact Datadog Technical Support for assistance.
```
